pages/docs/concepts/asyncapi-document/adding-operations.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Adding Operations
+weight: 60
+---
+
+In a messaging system, the term "operations" refers to the various methods by which messages are exchanged between participants or components. 
```

Comment:
In the introduction part of the document, it would be nice to include what operations also mean in AsyncAPI document, how they relate, how they work or used briefly. Doing so, the reader will have better understanding and more context

Reply:
added some more lined within the introduction